src/main/resources/swagger/api-docs.yaml

```diff
@@ -1695,6 +1695,131 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/resources/v2/{resourceTypeName}/{resourceId}/policies/{policyName}/memberPolicies/{memberResourceTypeName}/{memberResourceId}/{memberPolicyName}:
+    put:
+      tags:
+        - Resources
+      summary: Add a user to a policy by role on another resource
```

Comment:
nit: this could add multiple users (in fact, that's our use case). Maybe "Add a member policy to a parent policy" or similar?